### ISSUE TITLE
Allow an administrator to enable / disable application auto upgrades globally

### ISSUE THREAD

**Issue by mflorea** [xwikisas/application-licensing#109]
ATM it's only possible to disable / enable auto upgrades per application, from the Licenses administration section. At the UI level, the quickest solution would be to add a checkbox on the Licenses section. Administrators need to go there to generate / buy a license so they will notice the new checkbox.

**Comment by oanalavinia** [comment]
I had a discussion with @mflorea about what enable / disable could mean globally and this adds 2 usecases:
- when the auto upgrades are globally disabled, which is the default value, the admin would have the possibility to still enable the feature per application
- when the auto upgrades are globally enabled, the admin would have the possibility to disable this feature per application; this also means that any application installed in the future would be automatically upgraded until added to the blocklist.

Basically, the enable / disable would actually mean switching between using a blocklist (with everything else being enabled) or an allowlist (with everything else being disabled)
This implementation should also take into consideration https://github.com/xwikisas/application-licensing/issues/123, so the admin has a better (visual) understanding of what application are actually upgraded.
